examples/jf-control-app/commands/pairing/PairingCommand.cpp

```diff
@@ -842,6 +839,12 @@ void PairingCommand::OnDiscoveredDevice(const Dnssd::CommissionNodeData & nodeDa
     // Ignore nodes with closed commissioning window
     VerifyOrReturn(nodeData.commissioningMode != 0);
```

Comment:
Compare against to_underlying(Dnssd::CommissioningMode::kDisabled)

Reply:
Fixed in latest commit.